automation/jenkins_build.sh

```diff
@@ -2,18 +2,26 @@
 set -e
 
 # Jenkins build steps
-ESCAPED_BRANCH_NAME=$(echo $sourceBranch | sed 's/[^a-z0-9A-Z_.-]/-/g')
+export ESCAPED_BRANCH_NAME=$(echo $sourceBranch | sed 's/[^a-z0-9A-Z_.-]/-/g')
+BASE_IMAGE_VERSION=$(tar -cf - base-image | md5sum | awk -F " " '{print $1}')
```

Comment:
The hash will change whenever the metadata like mtime changes, which happens all the time as it's set to the time of checkout